packages/services/src/balances/sip10-balances.service.ts

```diff
@@ -63,15 +65,33 @@ export class Sip10BalancesService {
     };
   }
 
+  public async getSip10AccountBalance(
```

Comment:
It would be great if we can extend this to also offer token specific account balances + addresses (for SIP-10 and Runes). 

This would save us having to filter the balance arrays client side.